RELEASE.md

```diff
@@ -1,3 +1,40 @@
+# Release 2.2.0
+
+The 2.2 minor series tracks TensorFlow 2.2.
+
+## Features
+
+- Profile plugin now should be pip installed.
+- Debugger:
+  - Added support for alerts (#3285)
+  - Support FULL_HEALTH debug mode (#3343)
+  - Integration with ngrx storage (#3359)
+- Add S3_ENDPOINT variable (#3347)
+- Confirm that the connection to tensorboard works or change to localhost (#2371)
+- Update --reload_multifile_inactive_secs default to 24 hours (#3243)
+
+## TensorBoard.dev updates
+
+- Improvements to [TensorBoard.dev] support:
```

Comment:
Let’s drop this level of nesting. The “Improvements to TensorBoard.dev
support” introductory bullet is helpful when we have multiple relevant
entries under the “Features” heading, but under the “TensorBoard.dev
updates” heading it’s just superfluous.